app/src/configurable_auth.h

```diff
@@ -78,6 +82,21 @@ namespace scitt
       }
       return true;
     }
+
+    static void log_auth_error(
+      const std::shared_ptr<ccf::RpcContext>& ctx, std::string& error_reason)
+    {
+      // CCF returns any errors in the auth policy with a 401 status
+      CCF_APP_INFO(
+        "ClientRequestId={} Verb={} URL={} Status_code=401",
```

Comment:
```suggestion
        "ClientRequestId={} Verb={} URL={} Status=401",
```

We should use a consistent format here, see https://github.com/microsoft/scitt-ccf-ledger/blob/3ceb7d750f27e5ee8ce95207b30f8253919b6f51/app/src/tracing.h#L108.